src/components/utils/include/utils/logger/log4cxxlogger.h

```diff
@@ -30,28 +30,23 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-#ifndef SRC_COMPONENTS_INCLUDE_UTILS_PUSH_LOG_H_
-#define SRC_COMPONENTS_INCLUDE_UTILS_PUSH_LOG_H_
+#pragma once
 
 #include <log4cxx/logger.h>
-#include <string>
+#include "utils/ilogger.h"
 
 namespace logger {
 
-bool push_log(log4cxx::LoggerPtr logger,
-              log4cxx::LevelPtr level,
-              const std::string& entry,
-              log4cxx_time_t timeStamp,
-              const log4cxx::spi::LocationInfo& location,
-              const log4cxx::LogString& threadName);
+class Log4CXXLogger : public ThirdPartyLoggerInterface {
+ public:
+  Log4CXXLogger(std::string filename);
+  void Init() override;
```

Comment:
@VladSemenyuk `OVERRIDE` macro can be used instead

Reply:
What for?